libs/data_types/src/containers/account.hpp

```diff
@@ -3,7 +3,7 @@
 
 #include <cstdint>
 
-#include "sszpp/ssz++.hpp"
```

Comment:
`sszpp` installs its headers into directory `.../include/sszpp/`, with root directory specifying the name of the project. As far as I know, it's pretty standard way, instead of just placing all your headers into `include/`. So why should we remove "sszpp" here? And why is it even working without it? :)

Reply:
`sszpp` exports internal directory as its headers:
```cmake
# Create imported target sszpp
add_library(sszpp INTERFACE IMPORTED)

set_target_properties(sszpp PROPERTIES
  INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include/sszpp"
  INTERFACE_LINK_LIBRARIES "hashtree"
)
```
I have no idea how nix is able to build both variants